tensorboard/webapp/widgets/data_table/data_table_component.scss

```diff
@@ -20,13 +20,24 @@ $_accent: map-get(mat.get-color-config($tb-theme), accent);
 .data-table {
   border-spacing: 4px;
   font-size: 13px;
+  display: table;
   width: 100%;
 
-  th {
+  header,
```

Comment:
nit: I would prefer you put a class on the header element and then styled that

Reply:
Done. I also changed this to just be a div.